.docksal/commands/release

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+## Make a new release and deploys it.
+##
+## Usage: fin release
+
+# Abort if anything fails
+set -e
+
+# Backup the environments.
+echo -e "Backup Dev"
+fin terminus backup:create ${hostingsite}.dev --keep-for=365
+
+# Check Test enviroment exists
+# wc = word count. -l prints the number of lines.
+# tr = translating or deleting characters. -s replaces repeated characters.
+var_test_count=$(fin terminus env:list --field=id ${hostingsite} | grep 'test' | wc -l | tr -s " ");
+if [[ "$var_test_count" -eq "0" ]]; then
+  # Backup Test.
+  echo -e "Backup Test"
+  fin terminus backup:create ${hostingsite}.test --keep-for=365
+fi
+
+# Check Live enviroment exists
+var_live_count=$(fin terminus env:list --field=id ${hostingsite} | grep 'live' | wc -l | tr -s " ");
+if [[ "$var_live_count" -eq "0" ]]; then
+  # Backup Test.
+  echo -e "Backup Test"
+  fin terminus backup:create ${hostingsite}.live --keep-for=365
+fi
+
+# Get the latest release and up it by 1.
+# @TODO This strips anything after the last number like beta/alpha/rc.
+var_release_previous=$(gh release view)
+if [ -z "$var_release_previous" ]; then
+    var_tag_new=0.0.1
+else
+    var_tag_new=$(gh release view --json tagName --jq .tagName | awk -F. '{OFS="."; $NF+=1; print $0}')
```

Comment:
```suggestion
    var_tag_new=$(gh release view --json tagName --jq .tagName)
```

Reply:
| awk -F. '{OFS="."; $NF+=1; print $0}')

Adds the next patch number up from the latest release we fetched.